Mathlib/MeasureTheory/Measure/MeasureSpace.lean

```diff
@@ -618,7 +618,8 @@ theorem measure_liminf_eq_zero {s : ℕ → Set α} (h : (∑' i, μ (s i)) ≠
 #align measure_theory.measure_liminf_eq_zero MeasureTheory.measure_liminf_eq_zero
 
 theorem limsup_ae_eq_of_forall_ae_eq (s : ℕ → Set α) {t : Set α}
-    (h : ∀ n, s n =ᵐ[μ] t) : @limsup (Set α) ℕ _ s atTop =ᵐ[μ] t := by
+    (h : ∀ n, s n =ᵐ[μ] t) : limsup (α := Set α) s atTop =ᵐ[μ] t := by
+    -- Need to specify `α := Set α` above because of diamond; see gh issue #16932
```

Comment:
#16932 doesn't exist; is this https://github.com/leanprover-community/mathlib/pull/16932  ?

Reply:
Expanded

Reply:
Let's open a new issue for mathlib 4 with a lean 4 mwe, and reference that